reference/6/Microsoft.PowerShell.Management/Get-Content.md

```diff
@@ -3,185 +3,253 @@ ms.date:  06/09/2017
 schema:  2.0.0
 locale:  en-us
 keywords:  powershell,cmdlet
-online version:  http://go.microsoft.com/fwlink/?LinkId=821583
+online version:  http://go.microsoft.com/fwlink/?LinkID=113310
 external help file:  Microsoft.PowerShell.Commands.Management.dll-Help.xml
```

Comment:
These links to "online version" are supposed to be version specific. Many are broken but we need to leave these unchanged. I am working on a project to fix these link definitions.

Reply:
fixed